commands/channelargs_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func (s *MmctlUnitTestSuite) TestGetChannelArgs() {
+	s.Run("channel not found", func() {
+		notFoundChannel := "notfoundchannel"
+		notFoundErr := &model.AppError{Message: "channel not found", StatusCode: http.StatusNotFound}
+
+		s.client.
+			EXPECT().
+			GetChannel(notFoundChannel, "").
+			Return(nil, &model.Response{Error: notFoundErr}).
+			Times(1)
+
+		channels, err := getChannelsFromArgs(s.client, []string{notFoundChannel})
+		s.Require().Len(channels, 0)
+		s.Require().NotNil(err)
+		s.Require().Equal(fmt.Sprintf("1 error occurred:\n\t* channel %s not found\n\n", notFoundChannel), err.Error())
```

Comment:
nit: we can simplify a bit. This applies to other similar occurrences.
```suggestion
		s.Require().EqualError(err, fmt.Sprintf("1 error occurred:\n\t* channel %s not found\n\n", notFoundChannel))
```